### PR TITLE
Test distance metrics.

### PR DESCRIPTION
**Pull request by toregge**
@geirst : please review

### CODE REVIEW

File: tests/search/nearest_neighbor/distance_metrics.rb [hunk 1]
```diff
@@ -0,0 +1,131 @@
...
+  def feed_doc
+    doc = Document.new("distances", "id:#{@namespace}:#{@doc_type}::0").
+            add_field('euclidean', make_pos([1, 2])).
+            add_field('angular', make_pos([0, 2])).
+            add_field('prenorm', make_pos([0, 1])).
+            add_field('geodegrees', make_pos([0, 0])).
+            add_field('hamming', make_pos([3, 7])).
+            add_field('dotproduct', make_pos([2, 2]))
+    vespa.document_api_v1.put(doc)
+#    wait_for_hitcount('?query=sddocname:distances', 1)
```
Comment (geirst) [inline tests/search/nearest_neighbor/distance_metrics.rb:125]: Consider removing.

**Review by geirst — APPROVED** [review]
👍

File: tests/search/nearest_neighbor/distance_metrics.rb (hunk 1 above)
Comment (toregge) [reply tests/search/nearest_neighbor/distance_metrics.rb]: Done.

**Review by geirst — APPROVED** [review]
👍

**Pull request merged**